### ISSUE TITLE
Extension causes high cpu load

### ISSUE THREAD

**Issue by DJManas** [felixfbecker/vscode-php-intellisense#415]
[felixfbecker.php-intellisense-unresponsive.cpuprofile.txt](https://github.com/felixfbecker/vscode-php-intellisense/files/2886385/felixfbecker.php-intellisense-unresponsive.cpuprofile.txt)
- Issue Type: `Performance`
- Extension Name: `php-intellisense`
- Extension Version: `2.3.10`
- OS Version: `Darwin x64 18.2.0`
- VSCode version: `1.31.1`

:warning: Make sure to **attach** this file from your *home*-directory: `/Users/djmanas/felixfbecker.php-intellisense-unresponsive.cpuprofile.txt` :warning:

Find more details here: https://github.com/Microsoft/vscode/wiki/Explain:-extension-causes-high-cpu-load

**Comment by pitAlex** [comment]
Thats true, its the reason I remove it. Also there is no way to tell it exclude some files from checking.

**Comment by storm1er** [comment]
Duplicate of #411
